std_msgs/Header header
float32 angle_min_view
float32 angle_max_view
float32 angle_increment
float32[] ranges
float32[] angles
float32[] intensities
